README.md

```diff
@@ -127,4 +127,4 @@ If nvm installed: `nvm use` change version of current terminal to required.<br/>
 
 ### ARM Architecture support
 
-Install dependencies with `npm install --target_arch=x64` set target arch for running it through Rosetta (requires Rosetta installation).<br/>
\ No newline at end of file
+Install dependencies with `npm install --target_arch=x64` set target arch for running it through Rosetta (requires Rosetta installation).<br/>
```

Comment:
I think we don't need it anymore since repo uses `sass` instead of `node-sass`

Reply:
actually i have no idea why it is marked as a change, I didn't touch it and it doesn't seem like they are differ from each other